portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,113 @@
-#content {
+/* #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
 #greeting-container {
   margin-top: 20px;
-}
+} */
+
+/* Font */
+@import url("https://fonts.googleapis.com/css?family=Poppins:400,500,600,700,800");
+body {
+  font-family: "Poppins", sans-serif;
+  font-weight: normal;
+  font-style: normal;
+  color: #8a8fa3; }
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  color: #32333c;
+  margin: 0px; }
+
+h1 {
+  font-size: 60px; }
```

Comment:
The best practice to set font size is by using relative unit, e.g. %
https://blog.logrocket.com/the-elements-of-responsive-typography/

Reply:
Done by updating with responsive fonts using relative unit (% and rem).